visualization/app/codeCharta/ui/heightSettingsPanel/heightSettingsPanel.e2e.ts

```diff
@@ -1,20 +0,0 @@
-import { puppeteer, CC_URL } from "../../../puppeteer.helper"
-
-jest.setTimeout(10000)
-
-describe("HeightSettingsPanel", () => {
-	let browser, page
-
-	beforeAll(async () => {
-		browser = await puppeteer.launch()
-		page = await browser.newPage()
-	})
```

Comment:
plop anpassen

Reply:
Well, we (@NearW and me)decided to keep creating e2e test files, because it might encourage us to write more e2e tests. But when no test is written, we should delete the file.

Reply:
The idea here is to update the template, not remove it :)

Reply:
Ahh yes 👍 good point, but its officially the other PR/issue #616